src/vs/platform/terminal/common/capabilities/commandDetectionCapability.ts

```diff
@@ -600,6 +607,19 @@ class WindowsPtyHeuristics extends Disposable {
 		}
 	}
 
+	handleInput() {
```

Comment:
Good idea to adjust on input, we need to keep in mind that `onData` could happen when not in a prompt though. We should probably only do this when prompt start and command start has happened, but not command executed